src/app/api/admin/content/route.ts

```diff
@@ -140,7 +140,7 @@ export const POST = async (req: NextRequest) => {
         video_360p_mp4_3: metadata.video_360p_mp4_3,
         video_360p_mp4_4: metadata.video_360p_mp4_4,
 
```

Comment:
Change this to be `null` because the `/serivces/subtitle` route will always look for the videos will `null` subtitle link